definitions/variable/afolu/agriculture.yaml

```diff
@@ -1,156 +1,309 @@
+
+# Agricultural Trade
+- Agricultural Trade:
+    description: Net export of food, non-food and feed products 
+      (crops and livestock) and bioenergy crops (1st & 2nd generation)
+    unit: million t DM/yr
+    agmip: Production (PROD)
+    tier: 2
+    domain: AFOLU
+    subsubdomain: Agricultural Trade
+- Agricultural Trade|Crops:
+    description: Net export of crops
+    unit: million t DM/yr
+    agmip: Production (PROD) - All crops (CRP)
+    tier: 2
+    domain: AFOLU
+    subdomain: Agricultural Trade
+- Agricultural Trade|Crops|{Crop Types}:
+    description: Net export of {Crop Types}
+    unit: million t DM/yr
     agmip:
     tier: 2
+    domain: AFOLU
+    subdomain: Agricultural Trade
+- Agricultural Trade|Livestock:
+    description: Net export of livestock products (excluding fish)
+    unit: million t DM/yr
+    agmip: Production (PROD) - Livestock (LSP)
+    tier: 2
+    domain: AFOLU
+    subdomain: Agricultural Trade
+- Agricultural Trade|Livestock|{Livestock Products}:
+    description: Net export of {Livestock Products}
+    unit: million t DM/yr
+    agmip: Production (PROD) - Dairy (DRY)
+    tier: 2
+    domain: AFOLU
+    subdomain: Agricultural Trade
+- Agricultural Trade|Residues:
+    description: Net export of agricultural residues
+    unit: million t DM/yr
+    agmip:
+    tier: 2
+    domain: AFOLU
+    subdomain: Agricultural Trade
+
+# Crop Yields
+- Yield|Cropland|{Crop Types}:
+    description: Yield of cropland with {Crop Types}
+    unit: t DM/ha/yr
+    weight: Land Cover|Cropland
+    tier: 2
+    domain: AFOLU
+    subdomain: Crop Yields
+
+# Fertilizer Use -> see sdg-indicators for nitrogen fertilizer use (Nitrogen|Cropland Budget|Inputs)
+- Fertilizer Use|Phosphorus:
+    description: Phosphorus fertilizer use on cropland (synthetic and organic)
+    unit: Tg P/yr
+    tier: 2
+    domain: AFOLU
+    subdomain: Cropland Input
+
+# Agricultural Prices
+- Price|Agriculture|Food Products [Index]:
+    description: Farm-gate price index of food crops and livestock products (weighted average)
+    unit: Index (2020 = 1)
+    weight: Agricultural Production
+    navigate: Price|Agriculture|Non-Energy Crops and Livestock|Index
+    tier: 1
+    domain: AFOLU
+    subdomain: Agricultural Prices
+- Price|Agriculture|Food Products|Livestock [Index]:
+    description: Farm-gate price index of livestock products (weighted average)
+    sdg: 2
+    weight: Agricultural Production|Livestock
+    unit: Index (2020 = 1)
+    shape: Price|Agriculture|Livestock|Index
+    tier: 1
+    domain: AFOLU
+    subdomain: Agricultural Prices
+- Price|Agriculture|Food Products|Crops [Index]:
+    description: Farm-gate price index of food crops (weighted average)
+    unit: Index (2020 = 1)
+    weight: Agricultural Production|Crops
+    navigate: Price|Agriculture|Non-Energy Crops|Index
+    tier: 1
+    domain: AFOLU
+    subdomain: Agricultural Prices
+- Price|Agriculture|Food Products|Crops|Corn [Index]:
```

Comment:
Why is "Corn" not included in the list of crops for the "Agricultural Demand" and other variables?

Reply:
I renamed it to "Maize" for consistency. "Corn" is more commonly used in North America.